compiler/test/grainfmt/matches.input.gr

```diff
@@ -36,4 +36,4 @@ let testFormatting = charlist => {
       ['f', 'o', 'r', 'm', 'a', 't', ...rest] => true,
       _ => false
     }
-}
+}
```

Comment:
Why did this change?

Reply:
I accidentally formatted some of the inputs and thought I undid them properly but I suppose I didn't.